fishtest/fishtest/templates/base.mak

```diff
@@ -2,75 +2,47 @@
 <html>
 <head>
   <title>Stockfish Testing Framework</title>
-  <link href="/css/bootstrap.min.css" rel="stylesheet">
   <%block name="meta"/>
 
+  <link href="https://stackpath.bootstrapcdn.com/bootstrap/2.3.2/css/bootstrap.min.css" rel="stylesheet">
+  <link href="/css/application.css" rel="stylesheet">
+
   <script src="https://code.jquery.com/jquery-1.8.3.js"></script>
```

Comment:
(noobie writing here) if I'm not wrong we could update to
https://code.jquery.com/jquery-1.12.4.min.js

Reply:
yup, upgrading to v1.12.4 is safe. v3.4.1 drops support for old browsers like IE8, which we may not care about. You could check Google Analytics to see if any testers still use old browsers. I'll switch this to v3.4.1 unless we need to support them.